### DISCUSSION TITLE
How to disable this "Change password" popup in Chrome for testing?

### DISCUSSION THREAD

**Discussion by gokultw** [getgauge/taiko#2731] [General]
In earlier versions of Taiko, the Chromium browser is opened to executed the tests, in the latest version `Chrome for testing` browser is opened and after login, the "Change password" popup is displayed and without accepting this popup the clicks are not working in browser. How do I disable this popup or accept this popup?

**Comment by zabil** [comment]
@gokultw I've deleted the image because you just shared the info with a publicly reachable url exposing that it's using a breached password. Please change the password.

**Reply by gokultw** [reply]
<img width="428" alt="image" src="https://github.com/getgauge/taiko/assets/104197616/8aa010dc-e2ac-4e3c-9dd5-314e247cd9a1">
I have cropped the image @zabil

**Reply by zabil** [reply]
Can you try changing the password? That's the ideal way to get rid of this popup. 
Alternatively you can refer https://docs.taiko.dev/api/openbrowser/ for options to be passed from the list https://peter.sh/experiments/chromium-command-line-switches/
